common/app/routes/challenges/components/video/Video.jsx

```diff
@@ -58,13 +61,14 @@ export class Video extends React.Component {
   render() {
     const {
       title,
+      blockName,
       shouldShowQuestions
     } = this.props;
-
+    let titleString = challengeTitle({ blockName, title });
```

Comment:
Ok, sounds like it does not change per render but per challenge.

This is logic that should be in a selector instead, though. Saves a few calculations per render and is good practice.